template/deploy/helm/[[product]]-operator/Chart.yaml.j2

```diff
@@ -1,33 +1,11 @@
-# =============
-# This file is automatically generated from the templates in stackabletech/operator-templating
-# DON'T MANUALLY EDIT THIS FILE
-# =============
```

Comment:
Don't we want this header still?

Reply:
I believe the answer is "no" as it'll end up in the generated helm charts and it's more confusing than anything else there IMO

Reply:
Ok